invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/header.html

```diff
@@ -58,7 +58,7 @@
                     {%- else %}
                     <li class="navbar-item navbar-options">
 
-                    <li{{ ' class="navbar-item navbar-options active"'|safe if item.active and loop.depth == 1 else 'navbar-item' }}>
+                    <li{{ 'class="navbar-item navbar-options active"'|safe if item.active and loop.depth == 1 else 'navbar-item' }}>
```

Comment:
```suggestion
                    <li class={{ "navbar-item navbar-options active" if item.active and loop.depth == 1 else "navbar-item" }}>
```

otherwise `navbar-item'` is not assigned to the class...

Reply:
doing so seems to work with the spacing, but there is still an empty open  ```<li class="navbar-item navbar-options"></li>```
tag. which I removed to fix the ui showing an empty box onhover.

Reply:
Perfect thanks! The failing tests are on me. I am fixing them.